### PR TITLE
introduce templating for container port name

### PR DESCRIPTION
**Pull request by brandshaide**
fixing #972

### CODE REVIEW

**Comment by brandshaide** [comment]
I don't get the meaning of the linters error message 

```templates/vminsert-deployment.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 43: did not find expected comment or line break```

**Comment by brandshaide** [comment]
I don't get why the linter is reporting an YAML parse error: 

```YAML parse error on victoria-metrics-cluster/templates/vminsert-deployment.yaml: error converting YAML to JSON: yaml: line 42: did not find expected comment or line break```

File: charts/victoria-metrics-cluster/templates/vminsert-deployment.yaml [hunk 1]
```diff
@@ -73,7 +73,7 @@ spec:
           env: {{ toYaml . | nindent 10 }}
           {{- end }}
           ports:
-            - name: http
+            - name: "{{ .Values.vminsert.ports.name }}"  | default "http"
```
Comment (Haleygo) [inline charts/victoria-metrics-cluster/templates/vminsert-deployment.yaml:76]:
```suggestion
            - "{{ .Values.vminsert.ports.name | default "http" }}" 
```

**Review by Haleygo — COMMENTED** [review]
Thanks for the pull request!
Could you also help adding this for other components like vmselect and vmstorage?

**Comment by Haleygo** [comment]
LGTM!
Would you mind adding a line to the [changelog](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-cluster/CHANGELOG.md)?

**Comment by brandshaide** [comment]
> LGTM! Would you mind adding a line to the [changelog](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-cluster/CHANGELOG.md)?

thanks for your help! Of course I'll add this to the changelog

**Comment by brandshaide** [comment]
@Haleygo [here you go](https://github.com/VictoriaMetrics/helm-charts/pull/992)

**Review by Haleygo — APPROVED** [review]

**Comment by Haleygo** [comment]
Thank you for contribution!

File: charts/victoria-metrics-cluster/values.yaml [hunk 2]
```diff
@@ -473,6 +476,8 @@ vmstorage:
     # -- Image pull policy
     pullPolicy: IfNotPresent
   # -- Name of Priority Class
+  ports:
```
Comment (hanumanhuda) [inline charts/victoria-metrics-cluster/values.yaml:479]: isn't this need to be singular or it has to support multiple ports?

**Comment by hanumanhuda** [comment]
this needs to update the port name for liveness and readyness probe because it is using `http` as defualt.

Error as following
` Readiness probe errored: strconv.Atoi: parsing "http": invalid syntax`

**Pull request merged**